evm/deploy-gateway-v5.0.x.js

```diff
@@ -14,9 +14,9 @@ const readlineSync = require('readline-sync');
 const { Command, Option } = require('commander');
 const chalk = require('chalk');
 
-async function getAuthParams(config, chain) {
+async function getAuthParams(config, chainName, options) {
     printLog('retrieving addresses');
-    const { addresses, weights, threshold } = await getEVMAddresses(config, chain);
+    const { addresses, weights, threshold } = await getEVMAddresses(config, chainName, options.keyId, options.amplifier);
```

Comment:
```suggestion
    const { addresses, weights, threshold } = await getEVMAddresses(config, chainName, options);
```
pass in options to the helper too